src/locales/es.json

```diff
@@ -293,7 +293,8 @@
     "replay_video": "Reproducir su video grabado",
     "document_types": "Documentos que puede utilizar para verificar su identidad",
     "selfie_video_actions": "Acciones para grabar un video selfie",
-    "cross_device_verification": "Pasos requeridos para continuar con la verificación en su celular"
+    "cross_device_verification": "Pasos requeridos para continuar con la verificación en su celular",
```

Comment:
I think we use `móvil` instead of `celular`

Reply:
Updated the copy